packages/quantic/force-app/main/default/lwc/quanticSmartSnippetSuggestions/quanticSmartSnippetSuggestions.css

```
@@ -1,3 +1,7 @@
 .smart-snippet-suggestions__title {
   font-weight: 700;
 }
+
+.accordion__summary-heading {
+  --slds-c-accordion-heading-font-size: 1.1em;
```

We usually prefer REM in quantic.
```suggestion
  --slds-c-accordion-heading-font-size: 1.1rem;
```

---

noted ! :)